pkg/k8s/service_cache.go

```diff
@@ -397,7 +397,7 @@ func (s *ServiceCache) UniqueServiceFrontends() FrontendList {
 // all external endpoints if the service is marked as a global service. Also
 // returns a boolean that indicates whether the service is ready to be plumbed,
 // this is true if:
-// IF If ta local endpoints resource is present. Regardless whether the
+// IF If a local endpoints resource is present. Regardless whether the
```

Comment:
nit, there's three ifs in a row between these two lines :thinking:

Reply:
fixed.